app/services/workflow_executions/submission_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module WorkflowExecutions
+  # Service used to Prepare a WorkflowExecution
+  class SubmissionService < BaseService
+    def initialize(workflow_execution, wes_connection, user = nil, params = {})
+      super(user, params)
+      @workflow_execution = workflow_execution
+      @wes_client = Integrations::Ga4ghWesApi::V1::Client.new(conn: wes_connection)
```

Comment:
I gather that we will be using `Integrations::Ga4ghWesApi::V1::Client.new(conn: wes_connection)` in multiple services after this one. I think it would be better to have this in as a common method to which we can pass in the `wes_connection`. This way if the version changes we only need to update it in one place

Reply:
I agree, but let's see how the cards play first.

Reply:
Lol yeah sounds good!